src/OpenLoco/src/Windows/CompanyFaceSelection.cpp

```diff
@@ -75,19 +77,45 @@ namespace OpenLoco::Ui::Windows::CompanyFaceSelection
             self->initScrollWidgets();
             _9C68F2 = id;
             self->owner = id;
+
             const auto* skin = ObjectManager::get<InterfaceSkinObject>();
             self->setColour(WindowColour::secondary, skin->colour_0A);
-            _inUseCompetitors = CompanyManager::findAllOtherInUseCompetitors(id);
+
             self->rowCount = _numberCompetitorObjects;
             self->rowHover = -1;
             self->object = nullptr;
         }
+
+        // How will we be using the selected face?
+        _callingWindowType = callingWindowType;
+
+        // Make window blocking while open
```

Comment:
I appreciate these comments, I was wondering how you'd made a modal window and this lets lesser-experienced people like me learn these things in the openloco codebase <3